packages/components/icon/src/icon.js

```diff
@@ -39,12 +41,69 @@ export class TSIcon extends TSElement {
 			/** 90, 180, 270 */
 			rotate: { type: Number, reflect: true },
 			/** 'h', 'horizontal', 'v', 'vertical' */
-			flip: { type: String, reflect: true }
+			flip: { type: String, reflect: true },
+			/** INTERNAL svg content to be rendered inside icon */
```

Comment:
Wouldn't https://lit.dev/docs/components/properties/#internal-reactive-state be the better way to manage private state?